internal/handlers/common/params.go

```diff
@@ -132,9 +130,7 @@ func getBinaryMaskParam(value any) (types.Binary, error) {
 	case float64:
 		// TODO check float negative zero
 		if value != math.Trunc(value) || math.IsNaN(value) || math.IsInf(value, 0) {
-			return types.Binary{}, NewErrorMsg(ErrFailedToParse,
-				fmt.Sprintf("Expected an integer: $bitsAllClear: %#v", value),
-			)
+			return types.Binary{}, errNotWholeNumber
```

Comment:
The current value of `errNotWholeNumber` starts with a wrong function name. I think we could remove those prefixes if they are not correct

Reply:
Fixed